Assets/MixedRealityToolkit.Services/BoundarySystem/XR2018/MixedRealityBoundarySystem.cs

```diff
@@ -78,40 +61,60 @@ protected override List<Vector3> GetBoundaryGeometry()
             return boundaryGeometry;
         }
 
-        /// <summary>
-        /// Updates the <see href="https://docs.unity3d.com/ScriptReference/XR.TrackingSpaceType.html">TrackingSpaceType</see> on the XR device.
-        /// </summary>
+        /// <inheritdoc/>
```

Comment:
I intentionally didn't use `inheritdoc` here, because the message isn't the same. It references a specific Unity type that's only valid for this version of the system, which I thought added a bit more information for our documentation.